carshare/settings.py

```diff
@@ -265,3 +266,9 @@ def env_get_list(variable, default):
         },
     },
 }
+
+# contact details
+CONTACT_PHONE = '01720 575 301'
```

Comment:
Can you make these be read from the environment? e.g.
`CONTACT_PHONE = os.environ.get("CONTACT_PHONE", "01111 222 333")`

Then we can take actual IOSCV information out of the source code here and instead load it through our environment variable config instead.